### ISSUE TITLE
delete all application data (e.g., settings)

### ISSUE THREAD

**Issue by az0** [bleachbit/wishlist#7]
Delete everything, including configuration

**Comment by Tobias-B-Besemer** [comment]
I think such an option should be separate, in a separate cleaner, that is not shipped by default, be done. The risk that a user e.g. deletes all his mails when he "cleans up Thunderbird" is just to high! And the damage is e.g. by Firefox not so big, but maybe big enough if passwords & bookmarks after that are missing...
Also: There's a other wish open, to select all cleaners at ones... and if both come, I see the damage coming... ;-)
@az0, for which programs do you want such a cleaner ??? Maybe I can write you one... One cleaner were each option is a program that you can "reset"... ;-)

**Comment by az0** [comment]
I agree that the option should not be easy to enable, but my thought was this

1. The developer writes it in the normal CleanerML file, but it is marked in a special way. For example, `option id="nuclear"`.

2. The BleachBit application removes these from the normal list, and it puts them a separate menu under the menubar. This location has warnings

**Comment by Tobias-B-Besemer** [comment]
Sounds great!
